### DISCUSSION TITLE
custom page size with width and height not working

### DISCUSSION THREAD

**Discussion by gauravkatara** [diegomura/react-pdf#1286] [Q&A] [answered]
how to provide custom width and height of Xcm * Ycm to page.
I have tried using
<Page size={ width: "3cm", height: "6cm"} >

is there anything wrong with the syntax. Also i have tried providing simple number to height and width.

**Comment by diegomura** [comment, marked as answer]
Syntax is wrong. Try `<Page size={{ width: "3cm", height: "6cm"}} >` (double curly brackets)
